core/src/main/java/cz/xtf/core/bm/BuildManager.java

```diff
@@ -16,6 +18,23 @@ public BuildManager(OpenShift openShift) {
         if (openShift.getProject(openShift.getNamespace()) == null) {
             openShift.createProjectRequest();
             openShift.waiters().isProjectReady().waitFor();
+
+            try {
+                // Adding a label can be only done via 'namespace'. It cannot be set via 'project' API.
+                // Thus we do this separately. Also, to update namespace label, it's necessary to have 'cluster-admin'
```

Comment:
> it's necessary to have 'cluster-admin'

This is not entirely true. We need just one permission. Can you see this permission in exception? Can we document that one permission here in comment as well. 

I would expect something similar
```
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://api.ocp44-001-cz7i.dynamic.xpaas:6443/api/v1/namespaces/73-q31f/pods. Message: pods is forbidden: User "xpaasqe" cannot list resource "pods" in API group "" in the namespace "73-q31f". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=null, kind=pods, name=null, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=pods is forbidden: User "xpaasqe" cannot list resource "pods" in API group "" in the namespace "73-q31f", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).```

So permission is: `list resource "pods" in API group "" in the namespace`

Reply:
Otherwise looks good to me.

Reply:
Isn't `Error from server (Forbidden): namespaces "<name>" is forbidden: User "<user>" cannot patch resource "namespaces" in API group "" in the namespace "<name>"` what you mean?

Reply:
Yes it will be it. So permission will be `patch resource "namespaces"`. 

So can we reformulate sentence to something like:
`it's necessary to have permission to "patch resource namespaces", e.g. by having 'cluster-admin' role.`

Reply:
Oh, now I see. It is 2 lines below :)

Reply:
Yes :) but I updated per your suggestion anyway since you are right that mentioning `cluster-admin` this way could be misleading. Hope it's good now.